### PR TITLE
Update Build section of kroxylicious-sample/README.md

### PR DESCRIPTION
**Pull request by gracegrimwood**
### Type of change

- Documentation

### Description

Update Build section to use verify rather than install, add notes on building individual modules vs the whole project and on the `--also-make` flag.

### Additional Context

Addresses some points in https://github.com/kroxylicious/kroxylicious/issues/536 (not all of them, next up I'm taking a look at how we can add the dist profile into the kroxylicious-sample POM, as well as the possibility of running with the [Maven Exec Plugin](https://www.mojohaus.org/exec-maven-plugin/)), making the kroxylicious-sample README actually able to be followed successfully.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Review performance test results. Ensure that any degradations to performance numbers are understood and justified.
- [ ] Make sure all Sonar-Lint warnings are addressed or are justifiably ignored.
- [ ] Update documentation
- [ ] Reference relevant issue(s) and close them after merging
- [ ] For user facing changes, update CHANGELOG.md (remember to include changes affecting the API of the test artefacts too).

### CODE REVIEW

File: kroxylicious-sample/README.md [hunk 1]
```diff
@@ -8,27 +8,31 @@ This sample filter project provides examples to help you learn how [custom filte
 
 Building the sample project is easy! You can build the **kroxylicious-sample** jar either on its own or with the rest of the Kroxylicious project.
 
-To build all of Kroxylicious, including the sample:
+#### To build all of Kroxylicious, including the sample:
 
 ```
-$ mvn clean install
+$ mvn verify --also-make
```
Comment (SamBarker) [inline kroxylicious-sample/README.md:14]: I don't think we need `also-make` when building the entire reactor

File: kroxylicious-sample/README.md [hunk 2]
```diff
@@ -8,27 +8,31 @@ This sample filter project provides examples to help you learn how [custom filte
...
 ```
 
-To build the sample on its own:
+#### To build the sample on its own:
 
 ```
-$ mvn clean install -pl kroxylicious-sample
+$ mvn verify -pl :kroxylicious-sample --also-make
```
Comment (SamBarker) [inline kroxylicious-sample/README.md:20]: I think this is the only place we actually need to use `also-make`.

The other usages we are running the full build

**Review by SamBarker — APPROVED** [review]
A smooths a few of the rough edges off. Hopefully the next PR will address the rest

**Review by k-wall — APPROVED** [review]

File: kroxylicious-sample/README.md (hunk 1 above)
Comment (gracegrimwood) [reply kroxylicious-sample/README.md]: Very true, have fixed now.

**Review by SamBarker — APPROVED** [review]
LGTM

**Pull request merged**